tests/integration/skipping/skipping_test.cairo

```diff
@@ -0,0 +1,39 @@
+%lang starknet
+
+@external
+func test_skip() {
+    %{ skip(True, "AAA") %}
```

Comment:
`skip` should not be available in test functions directly, only in all setup hooks (both `setup_*` and `__setup__`). I don't want to pollute `TestExecutionEnvironment` logic with handling this cheatcode.

I know that slapping `skip` within test case would often be the least amount of code to write, but I suspect this would be a problematic thing in long term.

I think it might be high time to extract a `CommonTestCheatcodesFactory` class and not treat `SetupCheacodesFactory` as one.

Reply:
Done